packages/components/src/color-picker/stories/index.tsx

```diff
@@ -22,9 +25,13 @@ export default {
 		onChange: { action: 'onChange' },
 	},
 };
+export default meta;
 
-const Template = ( { onChange, ...props } ) => {
-	const [ color, setColor ] = useState();
+const Template: ComponentStory< typeof ColorPicker > = ( {
+	onChange,
```

Comment:
TS errors here, because it can't distinguish between `LegacyProps` or `ColorPickerProps`. Need to investigate that further.